packages/bundler/src/es6-rewriter.ts

```diff
@@ -432,4 +463,52 @@ export class Es6Rewriter {
         importSpecifier,
         {type: 'ImportSpecifier', imported: babel.identifier(exportName)});
   }
+
+  private _rewriteImportMetaToBundleMeta(
+      moduleFile: babel.File,
+      relativeUrl: FileRelativeUrl): babel.File {
+    // Generate a stand-in for any local references to import.meta...
+    // const __bundledImportMeta = {...__importMeta, url: __bundledImportURL};
+    const bundledImportMetaName = '__bundledImportMeta';
+    const bundledImportMetaDeclaration =
+        babel.variableDeclaration(  // leave this in for clang-format
+            'const',
+            [babel.variableDeclarator(
+                babel.identifier(bundledImportMetaName),
+                babel.objectExpression([
+                  babel.spreadProperty(babel.memberExpression(
+                      babel.identifier('import'), babel.identifier('meta'))),
+                  babel.objectProperty(
+                      babel.identifier('url'),
+                      babel.newExpression(
```

Comment:
We want to get the `.href` property of this new url, as `import.meta.url` is a string

Reply:
DERP!  Really good catch.  Fixed.